targets/targets.json

```diff
@@ -4323,6 +4323,42 @@
             "network-default-interface-type": "ETHERNET"
         }
     },
+    "MIMXRT1170_EVK": {
```

Comment:
Please override `supported_c_libs` to indicate only support for `GCC_ARM`.
If you also support the baremetal profile please override `supported_application_profiles`.

Reply:
I added this now